test/fixtures/unsupported-domain/domains.schema

```diff
@@ -0,0 +1,3 @@
+{
+  UnsupportedDomain: [],
```

Comment:
I'd suggest to rename this (and file changing appropriately) to 'IncorrectDomainDefinition'.